javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -688,7 +686,13 @@ protected IType createProtocolPagedRestResponseReturnType() {
      * @return method visibility, null if do not generate.
      */
     protected JavaVisibility methodVisibility(ClientMethodType methodType, boolean hasContextParameter) {
-        return VISIBLE;
+        if (JavaSettings.getInstance().isLowLevelClient()) {
+            return (methodType == ClientMethodType.SimpleAsync || methodType == ClientMethodType.SimpleSync)
+                    ? NOT_GENERATE
+                    : VISIBLE;
+        } else {
+            return VISIBLE;
+        }
```

Comment:
Two `VISIBLE` branches can be combined to one.

Reply:
Probably keep it, so it is easier to see what is configured for LLC and what is configure for default.